tests/wpt/metadata/FileAPI/url/url-in-tags.window.js.ini

```diff
@@ -1,4 +1,5 @@
 [url-in-tags.window.html]
+  expected: TIMEOUT
   [Blob URL fragment is implemented.]
-    expected: FAIL
+    expected: TIMEOUT
```

Comment:
This one is due to https://github.com/servo/servo/pull/20794/files#diff-f3bbe86981f8b2d223aa16ad8a140d13R45 where `iframe.onload` is changed to `iframe.contentWindow.onscroll`.